domain.yml

```
@@ -175,16 +176,16 @@ responses:
   - text: Sorry, {PERSON} is not in your list of known recipients.
   utter_insufficient_funds:
   - text: Sorry, you don't have enough money to do that!
-  utter_known_recipients:
-  - text: These are your known recipients {known_recipients}.
   utter_credit_card_balance:
   - text: The current balance for your {credit_card} account is {currency}{amount_of_money}.
-
+  utter_recipients:
+  - text: These are your known recpients to whom you can send money:\n{formatted_recipients}
```

Added double quotes because of the ':' being weird with YAML. 

```suggestion
  - text: "These are your known recipients to whom you can send money:\n{formatted_recipients}"
```